src/simplewallet/simplewallet.cpp

```diff
@@ -4468,6 +4485,23 @@ bool simple_wallet::show_transfer(const std::vector<std::string> &args)
       success_msg_writer() << "Timestamp: " << get_human_readable_timestamp(pd.m_timestamp);
       success_msg_writer() << "Amount: " << print_money(pd.m_amount);
       success_msg_writer() << "Payment ID: " << payment_id;
+      if (pd.m_unlock_time < CRYPTONOTE_MAX_BLOCK_NUMBER)
```

Comment:
I need to stop responding so quickly, that max line was just 2 lines below this (so it won't report as being on the genesis block). Still an inaccuracy without the special zero case (which is why initially flagged it). I am not sure how many people use the unlock timestamp feature. It probably isn't many, and when it does happen it is just an annoyingly incorrect message. And if the default is zero to signal `block_height + current` its not possible to know which case without using `boost::optional` for the "upgrade" path. Probably not worth it.